hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/s3guard/S3GuardTool.java

```diff
@@ -1607,6 +1610,7 @@ public int run(String[] args, PrintStream out) throws
       }
 
       final CommandFormat commandFormat = getCommandFormat();
+
```

Comment:
Should we add a check here for both flags being set at the same time and Fail if they are?

Reply:
sure. I'll do that. not exclusive

Reply:
yes. and here's why: let the user define the order in which he wants to run the checks.
I was thinking about the ordering of checks a bit, and the following seemed appropriate:
1. s3->ddb
2. ddb->s3
3. internal ddb

Reason: the first 2 can change the internal structure of the ddb but after the third one we can be sure that the store is left in a consistent state. 
But it's just a recommendation. Let's make the call exclusive.